src/sentry/quotas/base.py

```diff
@@ -419,6 +419,13 @@ def get_abuse_quotas(self, org):
                 categories=[DataCategory.METRIC_BUCKET],
                 scope=QuotaScope.ORGANIZATION,
             ),
+            AbuseQuota(
```

Comment:
Added this abuse quota to have additional control of the custom namespace in the future. This is not tied to the killswitch.